odbc/src/api/functions.rs

```diff
@@ -1272,14 +1259,13 @@ pub unsafe extern "C" fn SQLDriversW(
 /// # Safety
 /// Because this is a C-interface, this is necessarily unsafe
 ///
-#[named]
 #[no_mangle]
 pub unsafe extern "C" fn SQLEndTran(
     _handle_type: HandleType,
-    handle: Handle,
+    _handle: Handle,
     _completion_type: SmallInt,
 ) -> SqlReturn {
-    unimpl!(handle);
+    SqlReturn::SUCCESS
```

Comment:
Can you add a short comment explaining that even though we report that we don't support transactions, some tools still try to use them, and because we only offer read-only access, having SQLEndTran be a no-op is safe.